airbyte-commons-worker/src/main/java/io/airbyte/workers/RecordSchemaValidator.java

```diff
@@ -52,31 +62,55 @@ public RecordSchemaValidator(final FeatureFlagClient featureFlagClient,
       ((ObjectNode) schema).put("$schema", "http://json-schema.org/draft-07/schema#");
       validator.initializeSchemaValidator(stream.toString(), schema);
     }
-
   }
 
   /**
    * Takes an AirbyteRecordMessage and uses the JsonSchemaValidator to validate that its data conforms
    * to the stream's schema If it does not, this method throws a RecordSchemaValidationException
```

Comment:
nit: update this javadoc, we are no longer throwing a `RecordSchemaValidtionException`